go.mod

```diff
@@ -62,9 +74,9 @@ require (
 	google.golang.org/api v0.9.0
 	gopkg.in/robfig/cron.v2 v2.0.0-20150107220207-be2e0b0deed5
 	gopkg.in/yaml.v3 v3.0.0-20190709130402-674ba3eaed22
-	k8s.io/api v0.0.0-20190816222004-e3a6b8045b0b
-	k8s.io/apiextensions-apiserver v0.0.0-20190820064606-e49a3471dba5
-	k8s.io/apimachinery v0.0.0-20190816221834-a9f1d8a9c101
+	k8s.io/api v0.0.0-20190918195907-bd6ac527cfd2
+	k8s.io/apiextensions-apiserver v0.0.0-20190918201827-3de75813f604
+	k8s.io/apimachinery v0.0.0-20190817020851-f2f3a405f61d
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
 	k8s.io/code-generator v0.0.0-20190831074504-732c9ca86353
```

Comment:
I've updated this PR to pin Kubernetes dependencies to 1.15.3.

As you can see here, client-go and code-generator were already using this version, but apimachinery was not updated to do the same.

Because we previously had not regenerated clientsets using the 1.15.3 code-generator, despite bumping our dependency on code-generator correctly, we did not run into compilation issues.

We're now all in sync, and it is not possible to upgrade to 1.16.0 because the knative libraries depend on metav1.Initializers, which is removed in 1.16.